app/src/organisms/PipetteWizardFlows/constants.ts

```diff
@@ -33,8 +33,9 @@ export const CALIBRATION_PROBE = {
 export const HEX_SCREWDRIVER = {
   loadName: 'hex_screwdriver',
   displayName: HEX_SCREWDRIVER_DISPLAY_NAME,
+  //  TODO(jr, 4/3/23): add this subtitle to i18n
   subtitle:
-    'Provided with robot. Using another size can strip the instruments’s screws.',
+    'Provided with the robot. Using another size can strip the instruments’s screws.',
```

Comment:
```suggestion
    'Provided with the robot. Using another size can strip the instrument's screws.',
```
This typo shows up a lot in tests. Also, do we want straight or curly quote in the string?

Reply:
We use `’` instead of `'` for apostrophes in strings because we using the `'` around the string. There are probably more reasons why we use it. I think visually, they look the same in the UI since no one from design has mentioned it. If we want to make the switch over going over, I guess we _could_ but thats a conversation to be had with the team :D